buildroot-external/rootfs-overlay/etc/NetworkManager/NetworkManager.conf

```
@@ -5,7 +5,7 @@ autoconnect-retries-default=0
 rc-manager=file
 
 [keyfile]
-unmanaged-devices=type:bridge;type:tun;type:veth
+unmanaged-devices=type:bridge;type:tun;interface-name:veth*
```

According: https://developer.gnome.org/NetworkManager/stable/NetworkManager.conf.html#device-spec

```suggestion
unmanaged-devices=type:bridge;type:tun;driver:veth
```